src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs

```diff
@@ -213,15 +220,15 @@ public Task<ResultWrapper<ForkchoiceUpdatedV1Result>> Handle(ForkchoiceStateV1 f
         if (shouldUpdateHead)
         {
             _poSSwitcher.ForkchoiceUpdated(newHeadBlock.Header, forkchoiceState.FinalizedBlockHash);
-            if (_logger.IsInfo) _logger.Info($"Synced chain Head to  {newHeadBlock.ToString(Block.Format.Short)}");
+            if (_logger.IsInfo) _logger.Info($"Synced chain Head to {newHeadBlock.ToString(Block.Format.FullHashAndNumber)}");
```

Comment:
Wondering if should go other way and use `.Short` for both received new block and Synced and only use FullHash for errors? Is it useful?

Invalid block might want to copy paste the whole Hash; for valid blocks is 64 random hex bytes just noise?

Reply:
Dunno generally I hate short hashes when investigating any issue. It is acceptable if you have block number, but even then suboptimal.

Reply:
But now they align well with Received :)
So either both or none?

Reply:
Ok lets do both

Reply:
> Dunno generally I hate short hashes when investigating any issue. It is acceptable if you have block number, but even then suboptimal.

I think any errors should still use long hash

Reply:
But repeating the long hash lots and lots when is success seems like noise?